models/workoutlog.js

```diff
@@ -0,0 +1,20 @@
+// data for exercise
+
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const exerSchema = new mongoose.Schema({
+    exercise: {
+        type: String,
+        enum: ['Pushup', 'Pull Ups', 'Chin Ups', 'Barbell Squats', 'Air Squats', 'Bulgarian Split Squat', 'Goblet Squat','Dumbbell Squat', 'Wall Sits', 'Conventional Deadlift', 'Sumo Deadlift', 'Kettle Bells', 'Outdoor Walk', 'Indoor Walk', 'Outdoor Run', 'Treadmill', 'Row', 'Bicycle', 'Bench Press', 'Dips', 'Dumbbell Press', 'Dips']
+    }
+});
+
+const findExercise = module.exports = mongoose.model('exercise', exerSchema);
+
+
+// display 
+findExercise.find({}, function (err, result) {
+    if (err) return handleError(err);
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
    if (err) {return handleError(err);}
```